pkg/pool-linear/contracts/erc4626/ERC4626LinearPool.sol

```diff
@@ -67,29 +67,23 @@ contract ERC4626LinearPool is LinearPool {
         // whereas mainToken is DAI. But the 1:1 relationship holds, and
         // the pool is still valid.
 
-        // _getWrappedTokenRate is scaled e18, so we may need to scale totalAssets/totalSupply
+        // _getWrappedTokenRate is scaled e18, so we may need to scale IERC4626.convertToAssets()
         uint256 wrappedTokenDecimals = ERC20(address(wrappedToken)).decimals();
         uint256 mainTokenDecimals = ERC20(address(mainToken)).decimals();
-        uint256 digitsDifference = Math.add(18, wrappedTokenDecimals).sub(mainTokenDecimals);
+        uint256 digitsDifference = wrappedTokenDecimals.sub(mainTokenDecimals);
```

Comment:
assume it's true that `wrappedTokenDecimals` is always > `mainTokenDecimals`?

Reply:
Argh no, it's not always true. It's arbitrarily true in the test, so it wasn't caught. Thanks for noticing - I'll fix it